src/components/VLogoLoader/meta/VLogoLoader.stories.js

```diff
@@ -0,0 +1,53 @@
+import VLogoLoader from '~/components/VLogoLoader/VLogoLoader'
+
+export default {
+  component: VLogoLoader,
+  title: 'Components/VLogoLoader',
+  argTypes: {
+    status: {
+      default: 'idle',
+      options: ['loading', 'idle'],
+      control: { type: 'radio' },
+    },
+  },
+}
+
+const SimpleLoaderStory = (_, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  template: `
+    <div>
+      <VLogoLoader v-bind="$props" />
+      <p>Remember to test with the <code class="inline p-0"><pre class="inline bg-light-gray p-0 m-0 leading-normal">prefers-reduced-motion</pre></code> media query. You can find instructions for doing so in Firefox <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion#user_preferences">here.</a>
```

Comment:
Minor detail: the ending tag of the paragraph is missing.
```suggestion
      <p>Remember to test with the <code class="inline p-0"><pre class="inline bg-light-gray p-0 m-0 leading-normal">prefers-reduced-motion</pre></code> media query. You can find instructions for doing so in Firefox <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion#user_preferences">here.</a></p>
```